zebedee-cms/src/main/java/com/github/onsdigital/zebedee/model/Collection.java

```diff
@@ -1051,20 +1050,19 @@ public boolean deleteFileAndRelated(String uri) throws IOException {
      * newly created version is delete from the collection we need to tidy up these files as they can block users
      * from appoving the collection or adding the same content again.
      *
-     * @param uri
+     * @param targetURI
      * @return
      * @throws IOException
      */
-    private boolean deleteRelatedVersionedContent(String uri) throws IOException {
+    private boolean deletePreviousVersionFromReviewed(String targetURI) throws IOException {
         boolean deleteSuccessful = true;
-
-        List<String> versionedFiles = reviewedUris()
-                .stream()
-                .filter(contentURI -> contentURI.startsWith(contentURI) && isVersionedUri(contentURI))
```

Comment:
This is the origin of the defect.

It should be iterating through the collection reviewed content and finding any files that start with the URI of the file being deleted  `AND` are a versioned URI.

However this mistake is comparing the uri of each reviewed filed to its self. This will always evaluate `true` the result is it matches all previous version files in the reviewed instead of just the versions of the file being deleted.